src/vs/platform/keybinding/common/resolvedKeybindingItem.ts

```diff
@@ -29,6 +29,9 @@ export class ResolvedKeybindingItem {
 		}
 		this.bubble = (command ? command.charCodeAt(0) === CharCode.Caret : false);
 		this.command = this.bubble ? command!.substr(1) : command;
+		if (command === 'filesExplorer.paste') {
```

Comment:
Is this really needed? We cannot really depend on command IDs in this generic place. It looks like there is a concept of doing this when the command begins with `^`:

https://github.com/microsoft/vscode/blob/b206bc2c51df5d965b3dc1737ecdecdcfbf06766/src/vs/platform/keybinding/common/resolvedKeybindingItem.ts#L30